index.js

```diff
@@ -166,6 +166,23 @@ function showBadge() {
 }
 
 /**
+ * Purpose: Highlight a given item in the browser chrome
+ * @param {string} item - String of item you are attempting to highlight
+ */
+function highLight(item) {
+    var activeWindow = utils.getMostRecentBrowserWindow();
+        
+    UITour.getTarget(activeWindow, item, false).then(function(chosenItem) {
+        try {
+            UITour.showHighlight(activeWindow, chosenItem, "wobble");
+        }
+        catch(e) {
```

Comment:
`e` is defined here but never used. You could do `console.log('Could not highlight.....', e);`